### PR TITLE
Home load performance

### PR DESCRIPTION
**Pull request by jacobsimeon**
## Summary
This PR reduces the time it takes to load content for Home (i.e. the slate lineup). To achieve this improvement, we exclude the reader/offline content from the query. We fetch the details for a recommendation when a user selects the item and the reader is presented.

## References 
https://getpocket.atlassian.net/browse/IN-584

## Test Steps
* Launch app on home tab
* Notice the load time has improved

* Navigate to home tab
* Select a recommendation that is presented in the reader view (not web view)
* Notice the skeleton/loading screen is visible shortly while content is fetched from the server.

## PR Checklist:
- [x] Added Unit / UI tests
- [x] Self Review (review, clean up, documentation, run tests)
- [x] Basic Self QA

## Screenshots
loading-screen-iphone
loading-screen-wide

### CODE REVIEW

**Review by cyndichin — COMMENTED** [review]
Nice! Looks great in light mode, but not sure if we want to change dark mode from white to a grey color. Seems like a @moizsyed question. 
Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-08-03 at 14 27 53

File: PocketKit/Tests/PocketKitTests/Support/SavedItem+factories.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        let item = Item(context: space.context)
+        item.remoteID = remoteID
+        item.title = title
+        item.isArticle = isArticle
+        item.resolvedURL = resolvedURL
+        item.article = article
+
+        return item
+    }
+}
```
[the remark this answers is not in the document]
Comment (jacobsimeon) [reply PocketKit/Tests/PocketKitTests/Support/SavedItem+factories.swift:47]: yep, looks like I accidentally brought it back during a rebase

**Comment by jacobsimeon** [comment]
@cyndichin updated to address your comments.

Dark mode screenshot:
reader-skeleton-dark-mode

**Review by cyndichin — COMMENTED** [review]
Looks great! :)

**Review by cyndichin — APPROVED** [review]

**Pull request merged**